packages/functions/src/sendSMS/deliver.ts

```diff
@@ -106,3 +107,10 @@ export const deliverSMS = functions
       })
     )
   );
+
+export const deliverSMSTesting = functions
+  .region(__functionsZone__)
+  .firestore.document(
+    `${Collection.DeliveryQueues}/{organization}/${DeliveryQueue.EmailQueue}/{emailDoc}`
+  )
+  .onWrite(skippedDataTrigger("deliverEmailTesting", (data) => data?.payload));
```

Comment:
I don't understand the purpose of this trigger.

Reply:
Ok, I needed to read the code below. I now understand it. Sorry for the noise.

Reply:
Yeah...the main purpose: have a conditional export (have the export in `index.ts` have something to fall back to)...thinking about it now, maybe I could have used `null` as a fallback...

Reply:
On third thought, I believe this is not necessary. I'll open a PR to propose [these changes](https://github.com/eisbuk/EisBuk/commit/a1046df74a84461e1a8106268e001cab013943a7).

Reply:
@ikusteu Sorry for ignoring your comment. Github did not update the UI for me, so I missed it.